Repository clarification (#210)
bufbuild/docs.buf.build: docs/bsr/usage.md

The only requirement to push to the BSR is to have a module that [builds](../build/usage.md)
successfully, meaning that you should be able to run `buf build` successfully on your module.

> Before you can push a module into the BSR a repository must exist, owned by either a user or an organization, the user has access to. A repository can be created either through the UI or from the command line with `buf`.

Reviewer: Minor grammar change (removal of the comma after `organization`): ...owned by either a user or an organization the user has access to

> Before you can push a module into the BSR a repository must exist, owned by either a user or an organization, the user has access to. A repository can be created either through the UI or from the command line with `buf`.

Reviewer: ```suggestion
Before you can push a module into the BSR, a repository must exist and the user must own or have access to that repository. A repository can be created either through the UI or from the command line with `buf`.
```
